README.md

```diff
@@ -24,6 +38,7 @@ For Vega: [`vegaTooltip.vega(vgView[, options])`](docs/APIs.md#vgtooltip)
 ## Tutorials
 1. [Creating Your Tooltip](docs/creating_your_tooltip.md)
 2. [Customizing Your Tooltip](docs/customizing_your_tooltip.md)
+3. [Examples and Code on `bl.ocks.org`](https://bl.ocks.org/sirahd)
```

Comment:
I don't see anything wrong with having them link to my block though. Otherwise you'll have to update the markdown files every time new examples get added, rather than to my block directly